xbmc/addons/AddonBuilder.cpp

```diff
@@ -136,6 +138,8 @@ std::shared_ptr<IAddon> CAddonBuilder::Build()
       return CAudioEncoder::FromExtension(std::move(m_props), m_extPoint);
     case ADDON_AUDIODECODER:
       return CAudioDecoder::FromExtension(std::move(m_props), m_extPoint);
+    case ADDON_IMAGEDECODER:
+      return CImageDecoder::FromExtension(std::move(m_props), m_extPoint);
```

Comment:
There is maybe also the add to `FromProps¸ needed. Normally is the FromProps used for Repository contents and I'm think `CAddon` is enough, but completely not sure.